### PR TITLE
 RCT unpacker plus DQM 76X

### PR DESCRIPTION
**Pull request by nsmith-**
Re-make of  #10658
76X version of #10880

### CODE REVIEW

File: Configuration/StandardSequences/python/RawToDigi_cff.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def _modifyRawToDigiForStage1Trigger( theProcess ) :
     """
     Modifies the RawToDigi sequence if using the Stage 1 L1 trigger
     """
+    theProcess.load("EventFilter.RctRawToDigi.l1RctHwDigis_cfi")
```
Comment (davidlange6) [inline Configuration/StandardSequences/python/RawToDigi_cff.py]: @nsmith-  could you start a new cff that has all of the L1Trigger items in it? (to define L1RawToDigiSeq).

**Comment by nsmith-** [comment]
@davidlange6 after talking to @mulhearn I'm going to just drop this change to the StandardSequence for now, while he puts together a reorganization of that.
The DQM code defines its own unpacking sequence, so it will be unaffected

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
+1

**Comment by nsmith-** [comment]
Ok clearly the DAS error cannot be because of this PR

**Comment by davidlange6** [comment]
having the producer in the dqm sequence is even less desirable. when could we get a proper configuration? @mulhearn @nsmith- ?

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
please test

**Comment by deguio** [comment]
+1
seems reasonable to me now.

**Comment by mulhearn** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**